components/EmotionForm.js

```diff
@@ -138,7 +138,7 @@ const StyledFormContainer = styled.div`
   background-color: #e0e1f0;
   border: 1px solid #d3d3d3;
   border-radius: 0.5rem;
-  box-shadow: 0 2px 4px #000;
+  box-shadow: 0px 1px 4px #00000029;
```

Comment:
As mentioned, using custom variables for colors and shadows would save you time and effort  and create a more consistent design.  😻


For example, you could define `--shadow ` and reference them here instead of hardcoding values.